frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/MenuStrategyRemove/DialogStrategyRemove.tsx

```diff
@@ -227,7 +227,7 @@ const useOnSuggestRemove = ({
                 },
             });
             setToastData({
-                title: 'Changes added to the draft!',
+                text: 'Changes added to the draft!',
```

Comment:
sometimes we say "to draft" and sometimes "to the draft". Also extra exclamation mark.